test/r-apt/scenarios.json

```diff
@@ -1,69 +1,70 @@
 {
-	// "all-on-debian-stable": {
-	// 	"image": "debian:stable-slim",
-	// 	"features": {
-	// 		"r-apt": {
-	// 			"vscodeRSupport": "full",
-	// 			"installDevTools": true,
-	// 			"installREnv": true,
-	// 			"installRMarkdown": true,
-	// 			"installVscDebugger": true,
-	// 			"installJupyterlab": true,
-	// 			"installRadian": true
-	// 		}
-	// 	}
-	// },
-	// "all-on-focal": {
-	// 	"image": "ubuntu:focal",
-	// 	"features": {
-	// 		"r-apt": {
-	// 			"vscodeRSupport": "full",
-	// 			"installDevTools": true,
-	// 			"installREnv": true,
-	// 			"installRMarkdown": true,
-	// 			"installVscDebugger": true,
-	// 			"installJupyterlab": true,
-	// 			"installRadian": true
-	// 		}
-	// 	}
-	// },
-	// "all-on-jammy": {
-	// 	"image": "ubuntu:jammy",
-	// 	"features": {
-	// 		"r-apt": {
-	// 			"vscodeRSupport": "full",
-	// 			"installDevTools": true,
-	// 			"installREnv": true,
-	// 			"installRMarkdown": true,
-	// 			"installVscDebugger": true,
-	// 			"installJupyterlab": true,
-	// 			"installRadian": true
-	// 		}
-	// 	}
-	// },
-	// "all-on-not-testing": {
-	// 	"image": "debian:stable-slim",
-	// 	"features": {
-	// 		"r-apt": {
-	// 			"useTesting": false,
-	// 			"vscodeRSupport": "full",
-	// 			"installDevTools": true,
-	// 			"installREnv": true,
-	// 			"installRMarkdown": true,
-	// 			"installVscDebugger": true,
-	// 			"installJupyterlab": true,
-	// 			"installRadian": true
-	// 		}
-	// 	}
-	// },
-	"bspm-debian": {
-		"image": "mcr.microsoft.com/devcontainers/base:debian",
+	"all-on-debian-stable": {
+		"image": "debian:stable-slim",
 		"features": {
 			"r-apt": {
-				"installBspm": true
+				"vscodeRSupport": "full",
+				"installDevTools": true,
+				"installREnv": true,
+				"installRMarkdown": true,
+				"installVscDebugger": true,
+				"installJupyterlab": true,
+				"installRadian": true
 			}
 		}
 	},
+	"all-on-focal": {
```

Comment:
Why if we have jammy?  We provide the platform, not users so can we not just impose jammy?  May make testing and coverage easier...

Reply:
Simply drop the focal support.
https://github.com/rocker-org/devcontainer-features/blob/583387a2831c181905c571b306f3a976affbe5cb/src/r-apt/NOTES.md?plain=1#L5

I simply supported as much as I could support. If r2u currently does not work with focal, I will remove support for focal.

Reply:
As I recall, the default image (universal) for Codespaces was still focal based. So focal support is important.

Reply:
That is context I did not have -- and the reason we do of course support focal.

Google colab has focal too as default (or switched only very recently).   So yes if we reuse 'their' container focal support is good, and of course available.  Don't remove.

And let's maybe let 24 hours pass on PR reviews so that all timezones can chime in.

Reply:
> And let's maybe let 24 hours pass on PR reviews so that all timezones can chime in.

Sorry about this, but I merged it because this PR feature addition has no impact on existing installations and the release will make it easier for users to try out the new option.
(For example, @grantmcdermott can remove the Dockerfile from https://github.com/grantmcdermott/codespaces-r2u)

Reply:
Yes, I just spot-checked the repo by launching a new codespace and we still build from source on the first try. Sad.

Would be great if your help, and scipping of a Dockerfile, would improve upon that and get is the r2u easy that @grantmcdermott  intended to have when he set up the Dockerfile.  Which, as I understand it, we no longer need thanks your higher-level codespaces magic.  Yay!!

Reply:
I just tested with a devcontainer.json ([here](https://github.com/grantmcdermott/codespaces-r2u/blob/main/.devcontainer/devcontainer.json) fwiw) and everything works like a charm.

@eitsupi, you magician!